LiskHQ/lisk-desktop
# Add links to sign/verify message in menu in menu

### What issue have I solved?

-- #2613 

### How have I implemented/fixed it?

2 more links are added in this PR for the Vereify and Sign Message. Both are un the User Account dropdown menu, in the top bar.

NOTE: There is no design so far, we need icons for each link and order.

### How has this been tested?

1. Run the app.
2. Login using any network.
3. Check the User Account dropdown
4. Threre are 2 new butttons for verify and sign message, on click action should redirect to the correct page.


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
